### ISSUE TITLE
Displaying list of materials is broken

### ISSUE THREAD

**Issue by AGkizlis** [SilentChaos512/Silent-Gear#516]
## Versions

- **Silent Gear**: silent-gear-1.18.2-2.10.1.jar
- **Silent's Gems**: silents-gems-1.18.2-4.3.1.jar
- **Silent Lib**: silent-lib-1.18.2-6.2.0.jar
- **Forge**: 40.1.0
- **Modpack**: N/A
- **Optifine Installed**: Yes

## Expected Behavior
As described in the wiki, to get a list of materials for silent gear, you open JEI and search for "#material". Then, the items displayed in JEI are all the materials that can be used in silent gear.

## Actual Behavior
Searching for "#material" in JEI does not display all items that can be used in silent gear. Curiously enough, the ones missing (at least those I have noticed) are the silent gear ingots, namely blaze gold ingot, azure silver ingot, crimson iron ingot, azure electrum ingot, crimson steel ingot and tyrian steel ingot.

## Links/Images
Not needed I guess.

## Steps to Reproduce the Problem
1. Open JEI.
2. Search for "#material".
3. Look for tyrian steel. It's nowhere to be found.

**Comment by SilentChaos512** [comment]
Works fine for me. Maybe you didn't scroll down the item list enough? The Silent Gear materials show up after all the blueprints and templates (which unfortunately match the search because they have the word "material" in their tooltips.)

**Comment by AGkizlis** [comment]
I'll get back with some pics later. Just have to find someplace to host them.

**Comment by AGkizlis** [comment]
[Page 1/2 in JEI](https://im.ge/i/rWGUDP)
[Page 2/2 in JEI](https://im.ge/i/rWGhE1)
There are other materials missing too, but I guess fixing one will fix them all.
If there is anything else I could do to help you track down the problem, please ask.

**Comment by SilentChaos512** [comment]
Well, I can't get your images to load for some reason. You could just upload them here. But a log file would be more helpful.

**Comment by AGkizlis** [comment]
![image](https://user-images.githubusercontent.com/19387627/172223440-62dd50d9-eb9d-4f23-9610-baa1f33c787c.png)
![image](https://user-images.githubusercontent.com/19387627/172223507-457f1606-470c-4bea-9e57-e7fbe811db08.png)

As for the log file, I'm guessing you mean latest.txt
[latest.txt](https://justpaste.it/3ejot)

**Comment by SilentChaos512** [comment]
The log file shows 144 materials are being loaded from the server, which is a lot. No errors that I noticed. I have no idea. Is this still an issue for you?

**Comment by AGkizlis** [comment]
I have paused playing for a while, and decided to learn mod development myself. I'm going through some nice tutorials I found. If you don't mind, keep this issue open, and I'll answer you at some point. I may be able to find the cause by myself, which would be really nice practice I think.
As a first guess, I'd bet my money on JEI problem. Maybe the JEI version I use has some issues. I'll look into it and contact you here again.
Thanks for your interest.
